pkg/azurefile/controllerserver.go

```diff
@@ -1068,30 +1069,41 @@ func (d *Driver) copyFileShareByAzcopy(srcFileShareName, dstFileShareName, srcPa
 
 	jobState, percent, err := d.azcopy.GetAzcopyJob(dstFileShareName, authAzcopyEnv)
 	klog.V(2).Infof("azcopy job status: %s, copy percent: %s%%, error: %v", jobState, percent, err)
+
 	switch jobState {
 	case volumehelper.AzcopyJobError, volumehelper.AzcopyJobCompleted:
 		return err
 	case volumehelper.AzcopyJobRunning:
-		return fmt.Errorf("wait for the existing AzCopy job to complete, current copy percentage is %s%%", percent)
+		err = wait.PollImmediate(10*time.Second, time.Duration(d.waitForAzCopyTimeoutMinutes)*time.Minute, func() (bool, error) {
```

Comment:
If azcopy job state is Running, I think we can use 20 or 30 seconds since 10 seconds will print too many logs here